src/main/java/io/bobba/poc/communication/incoming/rooms/RequestFurniMovement.java

```diff
@@ -14,8 +20,21 @@ public void handle(GameClient client, ClientMessage request) {
         int rot = request.popInt();
 
         RoomUser user = client.getUser().getCurrentRoomUser();
+        RoomItem furni = user.getRoom().getRoomItemManager().getItem(itemId);
+        int room_id = user.getRoom().getRoomData().getId();
+
         if (user != null){
         	user.getRoom().getRoomItemManager().handleItemMovement(itemId, x, y, rot, user);
         }
+
+        try (Connection connection = BobbaEnvironment.getGame().getDatabase().getDataSource().getConnection(); Statement statement = connection.createStatement()) {
```

Comment:
This logic shouldn't be done here. RoomItemManager should have the responsibility or maybe a new class related with RoomItemManager